src/store.js

```diff
@@ -0,0 +1,48 @@
+import { defineStore } from "pinia";
+import { fetchBreeds } from "./data.js";
+export const useStore = defineStore("store", {
+  state: () => {
+    return {
+      breedsList: [],
+      searchList: [],
+      searchQuery: "",
+    };
+  },
+  getters: {
+    displayedBreeds(state) {
+      if (this.searchQuery) {
+        return this.searchList;
+      } else {
+        return this.breedsList;
+      }
+    },
+  },
+  actions: {
+    fetchData() {
+      // this.isLoading = true;
+      // this.message = "Data is loading...";
+
+      fetchBreeds()
+        .then((results) => {
+          // this.isLoading = false;
+          if (results.length > 0) {
+            this.breedsList = results.map((breed) => {
+              breed.isActive = false;
+              return breed;
+            });
+            console.log(this.breedsList);
+            // this.message = "";
+          } else {

```

Comment:
Dodal sem `displayedBreeds `getters, ki returna `breedsList `state oz `searchList `array, `fetchData `pa fetcha podatke iz `data.js` file in returna resultate v `breedsList `array.
`useStore `sem importov v `BreedsList `komponento, kjer sem potem znotraj `fetchData `metode, fetchov podatke iz `store.js`
Podatki se lepo naložijo na stran ko se app loada